tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -339,31 +339,24 @@ KEYPAD_NUMBERS.map((
 
 function f /* f */() {}
 function f(/* args */) {}
-function f() /* returns */ {
-}
-function f /* f */(/* args */) /* returns */ {
-}
+function f() /* returns */ {}
+function f /* f */(/* args */) /* returns */ {}
```

Comment:
This was necessary to align babylon and flow.. it was printing:

```js
// flow
function A() /*: T */ {}

// babylon
function A() /*: T */ {
}

Reply:
The fact that there was a newline here in the first place was just me being lazy :)